Tasks/AzureRmWebAppDeploymentV4/operations/AzureAppServiceUtility.ts

```diff
@@ -239,7 +254,7 @@ export class AzureAppServiceUtility {
         }: null;
     }
 
-    private _getNewMetadata(): any {
+    private _getNewMetadata(artifactAlias: string = null): any {
```

Comment:
Why null is a valid default? Is it when running inside a build?

Reply:
artifactAlias can be null when running inside build or when user has given package path as some file share location.